crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -134,7 +147,7 @@ where
         let height = block.header().height();
         let block_id = block.id();
         transaction
-            .storage::<FuelBlockIdsToHeights>()
+            .storage_as_mut::<FuelBlockIdsToHeights>()
```

Comment:
This change is not needed (?).

Not sure though, because I'm a little bit confused by both `fn storage()` and `fn storage_as_mut()` being equivalent.